src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilTest.java

```diff
@@ -341,58 +341,140 @@ public void testIsTypeDeclaration() {
     }
 
     @Test
-    public void testIsOfTypeTrue() {
+    public void testIsBooleanLiteralType() {
+        assertWithMessage("Result is not expected")
+                .that(TokenUtil.isBooleanLiteralType(TokenTypes.LITERAL_TRUE))
+                .isTrue();
+        assertWithMessage("Result is not expected")
+                .that(TokenUtil.isBooleanLiteralType(TokenTypes.LITERAL_FALSE))
+                .isTrue();
+        assertWithMessage("Result is not expected")
+                .that(TokenUtil.isBooleanLiteralType(TokenTypes.LOR))
+                .isFalse();
+    }
+
+    @Test
+    public void testAreOnSameLine() {
+        final int type = TokenTypes.COMPILATION_UNIT;
+        final DetailAstImpl astForTest = new DetailAstImpl();
+        final DetailAstImpl astForTest2 = new DetailAstImpl();
+        astForTest.setLineNo(type);
+        astForTest2.setLineNo(type);
+        final boolean result = TokenUtil.areOnSameLine(astForTest, astForTest2);
+
+        assertWithMessage("Token type did not match")
+            .that(result)
+            .isTrue();
+    }
```

Comment:
please remove unit tests which are not related to isOfType(). `TokenUtil.areOnSameLine` is already covered by other Check UTs.

Reply:
Changes Done